gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapperTest.java

```diff
@@ -367,8 +451,21 @@ private Bundle buildBundleWithStatementRefObservations() {
                 .setResource(new Observation().setId(STATEMENT_REF_ID_1)));
     }
 
+    private void assertConditionsMetaIsExpected(List<Condition> conditions, Meta expectedMeta) {
+        conditions.forEach(condition -> assertThat(condition.getMeta()).usingRecursiveComparison().isEqualTo(expectedMeta));
```

Comment:
I'd change this to be something like...

```suggestion
        assertAll(conditions.map(condition -> 
                () -> assertThat(condition.getMeta()).usingRecursiveComparison().isEqualTo(expectedMeta)
        ));
```

So we get the benefit of the `assertAll`